sdk/test/Services/RestXmlProtocol/Generated/Model/Internal/MarshallTransformations/XmlAttributesRequestMarshaller.cs

```diff
@@ -62,9 +62,7 @@ public IRequest Marshall(XmlAttributesRequest publicRequest)
             using (var xmlWriter = XmlWriter.Create(stringWriter, new XmlWriterSettings() { Encoding = System.Text.Encoding.UTF8, OmitXmlDeclaration = true, NewLineHandling = NewLineHandling.Entitize }))
             {   
                 xmlWriter.WriteStartElement("XmlAttributesRequest", "");
-                if(publicRequest.IsSetAttr())
-                    xmlWriter.WriteElementString("test", StringUtils.FromString(publicRequest.Attr));
-
+                xmlWriter.WriteAttributeString("test", "",StringUtils.FromString(publicRequest.Attr));
```

Comment:
Why is it no longer checking for `if(publicRequest.IsSetAttr())`?
Does it say to always write the attribute even with "" if it isn't set? It seems like if it wasn't set it shouldn't write the attribute.